packages/googledrive/configuration-schema.json

```diff
@@ -0,0 +1,36 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "baseUrl": {
```

Comment:
Hiya @aloundoye, So since we are not setting up `baseUrl`, or `username and password`. Please remove the from configuration-schema. 

See [googlesheet configuration-schema](https://docs.openfn.org/adaptors/packages/googlesheets-configuration-schema#full-schema) for example